app/controllers/users/new.js

```diff
@@ -22,55 +22,83 @@ export default Ember.Controller.extend({
     this.set("mobilePhone", "");
     this.set("email", "");
     this.set("position", "");
+    this.set("preferredPhone", "");
   },
 
-  formatMobileNumber(){
-    var mobile = this.get('mobilePhone');
-    if(mobile.length){
-      return config.APP.HK_COUNTRY_CODE + this.get('mobilePhone');
+  formatMobileNumber(mobileNumber) {
+    if (mobileNumber.length) {
+      return mobileNumber;
     } else {
-      return mobile;
+      return mobileNumber;
     }
   },
 
   actions: {
     saveUser() {
-      var loadingView = getOwner(this).lookup('component:loading').append();
-      var mobilePhone = this.formatMobileNumber();
-      var firstName = this.get('firstName');
-      var lastName = this.get('lastName');
-      var organisationId = this.get('organisationId');
+      var loadingView = getOwner(this)
```

Comment:
I think all `var` declarations can be replaced with `const.

Reply:
@Darshan-Chauhan Please review the PR I did the suggested changes